Exiled.Events/Patches/Events/Map/ExplodingFlashGrenade.cs

```diff
@@ -66,7 +66,7 @@ private static bool Prefix(FlashGrenade __instance)
             }
             catch (Exception exception)
             {
-                Log.Error($"Exiled.Events.Patches.Events.Map.ExplodingFlashGrenade: {exception}\n{exception.StackTrace}");
+                Log.Error($"{nameof(Handlers.Map.OnExplodingGrenade)} error: {exception}");
```

Comment:
Firstly, nameof() won't give us the full name, secondly, it's OnExplodingGrenade, thirdly, the format isn't the same as for other Transpilers/Prefixes

Reply:
True, I'll correct the first and second things.
But for the third one, doesn't `{exception}\n{exception.StackTrace}` just repeat the stack trace twice? Because by calling `exception.ToString()` you already get the stack trace.

Reply:
It does, so just do it that way https://github.com/Exiled-Team/EXILED/blob/2176b0e8e29dc9b3cac6988aabf6c6fd73d440a5/Exiled.Events/Patches/Events/Player/DequippedMedicalItem.cs#L38

Reply:
Ok, then we should change those methods that duplicate the stack trace, if any of them are still left.